### ISSUE TITLE
help with APA 7 mac

### ISSUE THREAD

**Issue by catrinita** [lx3h/LinguisticsJournals#1]
How can I install apa 7 on word 2015 macbook

**Comment by lx3h** [comment]
Thanks for your question. It is a good one. I personally do not own a Macbook, but I found a related [thread](https://github.com/codingo/BibWord) and it mentioned the location of the Style and Bibliography folders. Please let me know where yours are after you have succeeded.
Thanks.

**Comment by griffraff** [comment]
I have just installed this on Word for Mac V16.51, on Big Sur. The linked thread above is correct for adding the apa and styles file. 

"right-click on Microsoft Word 2008 and select show package contents. Put the files in:

Contents/Resources/Style/ On most Macs with Microsoft Word 2008 this will be: /Applications/Microsoft Office 2008/Microsoft Word.app/Contents/Resources/Style/"

For the Biblo file (that goes in 1033) I followed this: 

"Word templates are stored in your user Library folder, which is hidden by default. To open it, hold down the Option/Alt key while clicking on the Go menu and choosing Library.

Word 2011 templates are stored in ~/Library/Application Support/Microsoft/Office/User Templates/My Templates.

Word 2019 templates are stored in ~/Library/Group Containers/UBF8T346G9.Office/User Content/Templates."

from [here](https://answers.microsoft.com/en-us/msoffice/forum/all/word-for-mac-2019-templates/8462975f-0cfc-4341-b868-a02c685d0e7e) 

(edited because I spoke to soon)

I must not have things in the right spots, it changed the formatting of default types but I am not seeing the extra citation types and DOI isn't showing as a field.
